libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/store/EventStoreManager.java

```diff
@@ -235,6 +235,29 @@ public synchronized void setMaxEvents(int maxEvents) {
         this.maxEvents = maxEvents;
     }
 
+    /**
+     * Returns the maximum number of events that can be stored.
+     *
+     * @return Maximum number of events.
+     */
+    public int getMaxEvents() {
+        return maxEvents;
+    }
+
+    /**
+     * Returns number of stored events.
+     *
+     * @return Number of stored events.
+     */
+    public int getNumStoredEvents() {
```

Comment:
Added a couple of convenience APIs to return how many events can be stored and maximum number of events configured.